### DISCUSSION TITLE
Wrong Documentation Endpoint URLs?

### DISCUSSION THREAD

**Discussion by iantrudell** [strapi/strapi#7594] [Legacy Topics]
I've followed the [nginx Proxying Guide](https://strapi.io/documentation/v3.x/deployment/nginx-proxy.html) to setup Strapi in the "sub-folder split" configuration (API at /api, Admin Panel at /dashboard).

However when I try and use Swagger Docs to test out a REST query, the Swagger Docs still think the API is at the root instead of at /api and the query 404s.

How do I update the Documentation Plugin so that it understands the endpoints are all at /api instead of in the root?

**Comment by derrickmehaffy** [comment]
Interesting, let me test that @iantrudell it might be a bug in the documentation plugin. I will get back to you :)

**Reply by derrickmehaffy** [reply]
@iantrudell I tested locally and it's working fine, can you clarify where you had an issue?

If you are referring to the options in the swaggerUI to send requests you have to customize the settings manually for this.

https://strapi.io/documentation/v3.x/plugins/documentation.html#settings

```
  "servers": [
    {
      "url": "http://localhost:1337",
      "description": "Development server"
    },
    {
      "url": "YOUR_STAGING_SERVER",
      "description": "Staging server"
    },
    {
      "url": "YOUR_PRODUCTION_SERVER",
      "description": "Production server"
    }
  ],
```
